_posts/2024-03-06-happy-birthday.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Happy birthday OpenTTD!
+author: orudge
+---
+
+Exactly 21 years ago, I received an ICQ message (look it up, kids) from a guy named Ludvig Strigeus (nicknamed Ludde). "Hello, you probably don't know me,
+but I've been working on a project to clone TTD for a while." he said, more or less. He sent me a copy of what he'd been working on, and it was very exciting
+indeed. It was a fully functional version of Transport Tycoon Deluxe, written in C. Admittedly incomplete at that time, it was still quite remarkable. Ludde
+didn't want to release this to the public yet, and wasn't entirely sure what he was going to do with it. (He'd been working on it since 30th June 2002,
+so I guess that is technically the birthday of what would become OpenTTD...)
+
+<!-- more -->
+
+Ludde made more progress with the project over the coming year, and it looks like we even attempted some multiplayer games (not too reliable, especially
+over my dial-up connection at the time). Eventually, when he was happy with what he had created, he agreed to allow me to release the game as open
+source. Coincidentally, this happened exactly a year after I'd first spoken to him, on the 6th March 2004. But first, it needed a name. I don't think I
+thought very much about it, but decided that "OpenTTD" had a nice ring to it, so registered a SourceForge project, created an OpenTTD forum on TT-Forums,
+and [set it loose](https://www.tt-forums.net/viewtopic.php?f=29&t=6559). OpenTTD 1.0 was released. You can [still download it](https://cdn.openttd.org/openttd-releases/0.1.0/)
```

Comment:
1.0 -> 0.1.0?

Reply:
Oops. Changed it to 0.1 though, because at the time, it was simply [0.1](https://www.tt-forums.net/openttd/openttd-0.1.zip), not 0.1.0, and I like to be pedantic. :D